lib/run.js

```diff
@@ -32,10 +32,14 @@ function validateOpts(options) {
     
     validateNumberProp(options, 'rate');
     validateNumberProp(options, 'concurrent');
-    
-    // hack to enable concurrency mode,
-    // since forkmaster expects this to be defined
-    options.rate = options.rate || 0;
```

Comment:
I hate it when I do that and don't tell myself. I'll have to check it out.

GitKraken won't work on my home computer (same white screen and crash as if I were on a VPN), and I am too lazy to walk to my laptop.